package.json

```diff
@@ -8,17 +8,11 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@vue/cli-service": "^4.5.13",
+    "bootstrap-vue": "^2.17.3",
     "core-js": "^3.6.5",
-    "vue": "^2.6.11"
-  },
-  "devDependencies": {
-    "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-eslint": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.27.0",
-    "eslint-plugin-vue": "^7.9.0",
-    "vue-template-compiler": "^2.6.11"
+    "vue": "^2.6.11",
```

Comment:
this should be in ‘dependencies’, right?

Reply:
![image](https://user-images.githubusercontent.com/66675022/120075718-0c724000-c0c0-11eb-80b7-da6eda667d69.png)
it is

Reply:
its not showing here?

Reply:
@floydpink 
![image](https://user-images.githubusercontent.com/66675022/120075844-80144d00-c0c0-11eb-9470-f1ed266c1b32.png)
It's also there in my fork

Reply:
![image](https://user-images.githubusercontent.com/66675022/120075880-afc35500-c0c0-11eb-803b-9d68a841080f.png)
and it also shows up here, weird...